torch/_export/passes/lift_constants_pass.py

```diff
@@ -136,7 +145,7 @@ def lift_constants_pass(
                 # We already lifted this constant elsewhere. Just rewrite uses
                 # of this get_attr to point to the already-existing placeholder
                 # node.
-                const_placeholder_node = lifted_objs[constant_val]
+                const_placeholder_node = lifted_objs[constant_val][0]
```

Comment:
probably should change the pattern of constant_fqns.get then [0] to just `get_first_fqn` or something similar to avoid the redundancy.